src/benchmark.rs

```diff
@@ -106,9 +106,9 @@ impl PartialOrd for Benchmark {
 lazy_static! {
     static ref BENCHMARK_REGEX: Regex = Regex::new(r##"(?x)
         test\s+(?P<name>\S+)                        # test   mod::test_name
-        \s+...\sbench:\s+(?P<ns>[0-9,]+)\s+ns/iter  # ... bench: 1234 ns/iter
-        \s+\(\+/-\s+(?P<variance>[0-9,]+)\)         # (+/- 4321)
-        (?:\s+=\s+(?P<throughput>[0-9,]+)\sMB/s)?   # =   2314 MB/s
+        \s+...\sbench:\s+(?P<ns>[0-9,\.]+)\s+ns/iter  # ... bench: 1234 ns/iter
```

Comment:
The escape for `.` is superfluous here, since `.` isn't a meta character inside a character class. Please remove it.